BackendTests/UnitTests/MetAPIServiceTest.cs

```diff
@@ -0,0 +1,39 @@
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Backend.API.Services;
+using BackendTests.MockHelpers;
+using Xunit;
+
+namespace BackendTests.UnitTests
+{
+    public class MetAPIServiceTest
```

Comment:
These tests are not concerned with the current PR, and should be in a separate one

Reply:
Yes sir.